pom.xml

```diff
@@ -90,6 +90,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-validation</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-crypto</artifactId>
```

Comment:
```suggestion
            <artifactId>spring-boot-starter-security</artifactId>
```

Reply:
I don't need this dependency for this task. I think encryption is sufficient

Reply:
just don't forget to replace this with security starter when you will need more from this package, probably in the next HW